.github/actions/setup-integration-test/action.yml

```diff
@@ -34,41 +34,56 @@ runs:
         npm uninstall amazon-chime-sdk-js
         npm install ../../amazon-chime-sdk-js-$current_version.tgz
       shell: bash
-    - name: Create a Job ID
-      id: create-job-id
-      uses: filipstefansson/uuid-action@ce29ebbb0981ac2448c2e406e848bfaa30ddf04c
-    - name: Set JOB_ID Env Variable
-      run: echo "JOB_ID=${{ steps.create-job-id.outputs.uuid }}" >> $GITHUB_ENV
-      shell: bash
-    - name: Echo Job ID
-      run: echo "${{ steps.create-job-id.outputs.uuid }}"
-      shell: bash
     - name: Configure AWS Credentials
       uses: aws-actions/configure-aws-credentials@v4
       with:
         role-to-assume: ${{ inputs.aws-role-to-assume }}
         role-session-name: ${{ inputs.aws-role-session-name }}
         aws-region: us-east-1
+    - name: Setup Node.js - 20.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 20.x
+    - name: Check if SauceLabs credentials are provided
+      id: check-sauce
+      run: |
+        if [ -n "${{ inputs.sauce-username }}" ] && [ -n "${{ inputs.sauce-access-key }}" ]; then
```

Comment:
So since the inputs are now false, should this just echo `use_sauce=false` . If this is getting used elsewhere then above input update needs to be checked if fine.

Reply:
We plan to use this with daily test so next PR will update browser compatibility test which will pass sauce key